instances.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 IBM Corp.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kp
+
+import (
+	"context"
+	"time"
+)
+
+const (
+	dualAuthDelete = "dualAuthDelete"
+)
+
+// DualAuthPolicy represents a dual auth delete policy of a key as returned by the KP API.
+// this policy enables dual authorization for deleting a key
+type Policy struct {
+	CreatedBy  string     `json:"createdBy,omitempty"`
+	CreatedAt  *time.Time `json:"creationDate,omitempty"`
+	UpdatedAt  *time.Time `json:"lastUpdated,omitempty"`
+	UpdatedBy  string     `json:"updatedBy,omitempty"`
+	PolicyType string     `json:"policy_type,omitempty"`
+	PolicyData struct {
+		Enabled bool `json:"enabled,omitempty"`
+	} `json:"policy_data,omitempty" mapstructure:"policyData"`
+}
+
+// GetPolicy retrieves all policies by Key ID.
+func (c *Client) GetInstancePolicies(ctx context.Context) (interface{}, error) {
+	policyresponse := Policies{}
+
+	req, err := c.newRequest("GET", "instance/policies", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = c.do(ctx, req, &policyresponse)
+	if err != nil {
+		return nil, err
+	}
+
+	return policyresponse.Policies, nil
+}
+
+// SetPolicy updates a policy resource by specifying the ID of the key and either the rotation interval or dual auth or both .
```

Comment:
```suggestion
// SetInstancePolicies updates an instance-level policy
```
Or some other updated description for this function